pom.xml

```diff
@@ -89,28 +98,13 @@
       <resource>
         <directory>src/main/resources</directory>
         <filtering>true</filtering>
-        <includes>
```

Comment:
Why did you remove these? I thought that it was maybe better to list explicitly which files we're filtering, though, granted, it is a little more difficult with the new Cube capability to use maven-like properties in annotations.

Reply:
Some boosters have specific files such as keycloak.json, arquillian.xml etc. I think it should be safe to filter all files for the majority of the boosters. It would be quite hard to maintain the full list of inclusions.

Reply:
We can always exclude files in the boosters if we see an issue, but so far there were issues with not included patters only.